### PR TITLE
Add english translation and add app auto reloading after a config changement

### PR DESCRIPTION
**Pull request by NastuzziSamy**
No more `PureComponent` usage !

69242583_789378108125934_8838908421605425152_n

67848848_2392987184246153_2503509060716855296_n

### CODE REVIEW

**Comment by NastuzziSamy** [comment]
Changed to settings !

File: src/locales/en.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+		"AppLoader": {
+			"loading": "Application loading...",
+			"connect_cas": "CAS-UTC reconnection...",
+			"reconnection": "Reconnection...",
+			"reset_data": "Wipping data...",
+		},
+		"Home": {
+			"title": "Home",
+			"hi": "Hi",
+			"your_balance": "Hi %{name}, your solde :",
```
Comment (MercierCorentin) [inline src/locales/en.json]: I would have said `balance` like on Paypal

File: src/locales/en.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+			"no_balance": "Impossible to get your solde :(",
```
Comment (MercierCorentin) [inline src/locales/en.json]: Also `balance`

File: src/locales/en.json [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+			"week_positive": "credited this week.",
+			"week_negative": "spent this week."
+		},
+		"Refill": {
+			"title": "Refill",
+			"amount": "Refill amount :",
+			"pay": "Access to payment...",
+			"bad_amount": "The amount must be between %{min} and %{max}.",
+			"amount_error": "The amount is incorrect.",
+			"refill_checks": "Vérfication before refilling...",
```
Comment (MercierCorentin) [inline src/locales/en.json]: `Verification`

File: src/locales/en.json [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+			"recipient": "Recipient :",
+			"recipient_placeholder": "Search a recipient...",
+			"amount": "Amount of the transfer :",
+			"message": "Message : ",
+			"message_placeholder": "A (kind) message",
+			"transfer_button": "Transfer...",
+			"bad_amount": "The amount must be between %{min} and %{max}.",
+			"recipient_not_found": "No recipient found.",
+			"amount_error": "The amount is incorrect.",
+			"transfer_checks": "Vérfication before transfering...",
```
Comment (MercierCorentin) [inline src/locales/en.json]: Also `Verification`

File: src/locales/en.json [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+			"transfering": "Transfering...",
+			"transfer_confirmed": "Transfere of %{amount} confirmed to %{name}",
```
Comment (MercierCorentin) [inline src/locales/en.json]: `Transfer`

**Pull request merged**